api/v1/openapi.yaml

```diff
@@ -1754,9 +1754,28 @@ definitions:
         - Probe
         - Partial
       devices:
-          type: array
-          items:
-            type: string
+        description: |-
+
+
+          +k8s:deepcopy-gen=true
+        type: array
```

Comment:
Was there a reason for these extra whitespaces? We should followup with some documentation on our API fields.

Reply:
> Was there a reason for these extra whitespaces?

@aanm might have an answer.

> We should followup with some documentation on our API fields

IMHO, the added fields are self-explanatory.

Reply:
It's to avoid having the documentation of this fields starting with `+k8s:deepcopy-gen=true`